bundle/manifests/cert-manager.clusterserviceversion.yaml

```diff
@@ -60,6 +60,8 @@ spec:
                             fieldPath: metadata.name
                       - name: OPERATOR_NAME
                         value: cert-manager-operator
+                      - name: RELATED_IMAGE_CERT_MANAGER
```

Comment:
Can we keep it consistent with other operators of ours, and just name this `IMAGE` see https://github.com/openshift/cluster-kube-controller-manager-operator/blob/master/manifests/0000_25_kube-controller-manager-operator_06_deployment.yaml#L53-L54 or https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/manifests/0000_20_kube-apiserver-operator_06_deployment.yaml#L53-L54

Reply:
The Cert Manager Operator is going to be released thought CPaaS product pipeline (not ART). Having this in mind, we need to use `RELATED_IMAGE_*` variables as described in the link provided in the PR description.